tanglescope/statscollector/stats/noop.hpp

```diff
@@ -11,12 +11,12 @@ namespace statscollector {
 
 class NoopTXStats : public TXStats {
  public:
-  virtual void trackNewTX(iri::TXMessage&, PrometheusCollector::CountersMap& metrics){};
-  virtual void trackReattachedTX(PrometheusCollector::CountersMap& metrics){};
-  virtual void trackNewBundle(PrometheusCollector::CountersMap& metrics){};
-  virtual void trackConfirmedBundle(int64_t totalValue, uint64_t size, uint64_t avgBundleDuration,
-                                    PrometheusCollector::CountersMap& counters,
-                                    PrometheusCollector::HistogramsMap& histograms){};
+  virtual void trackNewTX(iri::TXMessage&, PrometheusCollector::CountersMap& /* metrics */){};
```

Comment:
Can you document these functions as we do here for example:
https://github.com/iotaledger/entangled/blob/develop/ciri/core.h

Reply:
Sure, add doc. at #91c8486

Reply:
please add directions in parameters as well, see https://github.com/iotaledger/entangled/blob/develop/ciri/api/api.h

Reply:
Thank you for pointing this out. Modified at https://github.com/iotaledger/entangled/pull/1297/commits/092494dbcb4156b95d4c4ddbb22b6ff058dd5a44

Reply:
Modified the func. dir. at https://github.com/iotaledger/entangled/pull/1297/commits/3beeafb0071d77cae0153e14a821a72fbc1b9aa7